clarify rc usage (#370)
cdr/docs: changelog/index.md

- Release _candidates_ begin appearing on the **second Wednesday** of each month
  (Coder may create multiple release candidates prior to the actual release)

> > We do not provide documentation for release candidates, and they should not be

Reviewer: ```suggestion
> We do not provide documentation for release candidates (any version containing `-rc`), and they should not be
```

> > used unless you've been asked to use it by Coder.

Reviewer: ```suggestion
> used unless you've been instructed to use them by Coder.
```

Reviewer 2: "unless the Coder team instructs you to do so" or something like that?
